src/utils/self_test/SConscript

```diff
@@ -9,24 +9,40 @@ import os
 import daos_build
 
 SELF_TEST = 'self_test.c'
+LIB_CONFIG_INI = 'src/configini.c'
+LIB_CONFIG_CNF = 'etc/selftest.cnf'
+LIB_CONFIG_ECNF = 'etc/selftest_expected.cnf'
+
+# Function to filter out gcc flags from this compilation env
+def remove_certain_gcc_flags(str):
+    import re
+    r = re.compile("\-Wframe\-larger\-than\=")
+    if r.match(str):
```

Comment:
(pylint-no-else-return) Unnecessary "else" after "return"

Reply:
Where is the `else` this pylint message is referring to?  Is this comment out-of-date?

Reply:
Not an old comment.  modified code to:
    if r.match(_str):
        return False
    else
        return True